packages/jest-util/src/prepareStackTrace.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+// eslint-disable-next-line max-len
+const FILTER_NOISE = /(\/|\\)jest(-.*?)?(\/|\\)(vendor|build|node_modules|packages)(\/|\\)/;
+
+const filterNoise = (
+  filename: ?string = '',
+  index: number,
+  patterns: Array<RegExp>,
+) => index !== 0 && patterns.map(pattern => pattern.test(filename || ''));
+
+const getCallSiteData = (callsite: CallSite) => {
```

Comment:
This is something that we could actually export by assigning it to some error proprety, eg:
```js
Error.callSiteData = getCallSiteData(callsite);
```